src/test/regress/expected/multi_modifying_xacts.out

```diff
@@ -1186,11 +1186,13 @@ COPY reference_failure_test FROM STDIN WITH (FORMAT 'csv');
 ERROR:  connection error: localhost:57637
 COMMIT;
 -- show that no data go through the table and shard states are good
+SET client_min_messages to 'ERROR';
```

Comment:
In pg12 the order of workers changed, so that we get a warning about the select failing on the first worker before the second worker gives us the correct result. In pg11 we happen to hit the healthy worker first, but would warn if we changed the order & tried the unhealthy worker first